doc/contribute/contributor_expectations.rst

```diff
@@ -305,10 +305,23 @@ Reviewer Expectations
   they address all non-blocking comments. PR authors should acknowledge every
   review comment in some way, even if it's just with an emoticon.
 
+- Style changes that the reviewer disagrees with but that are not documented as
+  part of the project can be pointed out as non-blocking, but cannot constitute
+  a reason for a request for changes. The reviewer can optionally correct any
+  potential inconsistencies in the tree, document the new guidelines or rules,
+  and then enforce them as part of the review.
+
+- Whenever requesting changes, reviewers must be able to point out the
+  corresponding guideline, rule or rationale in the project's documentation.
```

Comment:
*including past rulings made at architecture or TSC meetings

Reply:
Past rulings should not be included:
1. They are not readily available,
2. They are specific to a case and cannot be generalized without review of the rule,

Past rulings should undergo a review before being added to the documentation as a general rule, this allows a broader audience to verify that this general rule can be applied in all cases.

Reply:
If a past ruling applies to one specific thing like "this new subsystem is allowed" then it will say that in the minutes, if it applies to all rulings like "commits must be squashed properly" then it should apply to all, if someone is going to request changes then it is up to them to provide a link and excerpt from the minutes to substantiate that, and the architecture/TSG rulings are binding, as per their project roles, so yes this absolutely should remain here

Reply:
Everything is changing, past rulings can be revoked, no longer be applicable or detailed further. Whatever ruling that has been set in the past could no longer be valid. When a new subsystem is allowed it could later down the road be removed, the same is valid for general rulings. "General" rulings should be added to the documentation and removed or detailed when needed. "Specific" rulings (this new subsystem is allowed) should stay specific and cannot be used as a generalized ruling.

Reply:
past decisions should be documented somewhere when the decision was made. An exception made at one point does not mean it becomes a rule unless it is also documented. Maybe provide some examples of such "past rulings" and look into what those are not doucmented guidelines yet.